src/Assertion/AssertionContainerInterface.php

```diff
@@ -19,29 +19,9 @@
 
 declare(strict_types=1);
 
-namespace ZfcRbac\Role;
+namespace ZfcRbac\Assertion;
 
-use Zend\ServiceManager\AbstractPluginManager;
-use Zend\ServiceManager\Factory\InvokableFactory;
-use ZfcRbac\Container\ObjectRepositoryRoleProviderFactory;
-
-/**
- * Plugin manager to create role providers
- *
- * @method RoleProviderInterface get($name)
- *
- * @author  Michaël Gallego <mic.gallego@gmail.com>
- * @license MIT
- */
-final class RoleProviderPluginManager extends AbstractPluginManager
+interface AssertionContainerInterface
 {
-    protected $instanceOf = RoleProviderInterface::class;
-
-    /**
-     * @var array
-     */
-    protected $factories = [
-        InMemoryRoleProvider::class         => InvokableFactory::class,
-        ObjectRepositoryRoleProvider::class => ObjectRepositoryRoleProviderFactory::class,
-    ];
+    public function get($name): AssertionInterface;
```

Comment:
typehint string

Reply:
I can't because of `AbstractPluginManager::get($name, array $options = null)`